invenio_rdm_records/records/mappings/os-v1/rdmrecords/records/record-v7.0.0.json

```diff
@@ -0,0 +1,1491 @@
+{
+  "mappings": {
+    "dynamic_templates": [
+      {
+        "pids": {
+          "path_match": "pids.*",
+          "match_mapping_type": "object",
+          "mapping": {
+            "type": "object",
+            "properties": {
+              "identifier": {
+                "type": "text",
+                "fields": {
+                  "keyword": {
+                    "type": "keyword",
+                    "ignore_above": 256
+                  }
+                }
+              },
+              "provider": {
+                "type": "keyword"
+              },
+              "client": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      },
+      {
+        "parent_pids": {
+          "path_match": "parent.pids.*",
+          "match_mapping_type": "object",
+          "mapping": {
+            "type": "object",
+            "properties": {
+              "identifier": {
+                "type": "text",
+                "fields": {
+                  "keyword": {
+                    "type": "keyword",
+                    "ignore_above": 256
+                  }
+                }
+              },
+              "provider": {
+                "type": "keyword"
+              },
+              "client": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      },
+      {
+        "i18n_title": {
+          "path_match": "*.title.*",
+          "unmatch": "(metadata.title)|(metadata.additional_titles.title)",
+          "match_mapping_type": "object",
+          "mapping": {
+            "type": "text",
+            "fields": {
+              "keyword": {
+                "type": "keyword",
+                "ignore_above": 256
+              }
+            }
+          }
+        }
+      }
+    ],
+    "dynamic": "strict",
+    "date_detection": false,
+    "numeric_detection": false,
+    "properties": {
```

Comment:
for which fields did you apply the new analysers ?

Reply:
It's the titles, additional titles, description, and name sub fields of creators, contributors and person_or_org
Here is the diff for the record index:
```diff
@@ -613,7 +613,9 @@
                     "type": "keyword"
                   },
                   "name": {
-                    "type": "text"
+                    "type": "text",
+                    "analyzer": "accent_analyzer",
+                    "search_analyzer": "accent_analyzer"
                   },
                   "identifiers": {
                     "properties": {
@@ -682,7 +684,9 @@
                     "type": "keyword"
                   },
                   "name": {
-                    "type": "text"
+                    "type": "text",
+                    "analyzer": "accent_analyzer",
+                    "search_analyzer": "accent_analyzer"
                   },
                   "identifiers": {
                     "properties": {
@@ -715,7 +719,9 @@
                     }
                   },
                   "name": {
-                    "type": "text"
+                    "type": "text",
+                    "analyzer": "accent_analyzer",
+                    "search_analyzer": "accent_analyzer"
                   },
                   "type": {
                     "type": "keyword"
@@ -768,7 +774,9 @@
             }
           },
           "description": {
-            "type": "text"
+            "type": "text",
+            "analyzer": "accent_analyzer",
+            "search_analyzer": "accent_analyzer"
           },
           "additional_descriptions": {
             "properties": {
@@ -1100,10 +1108,15 @@
           },
           "title": {
             "type": "text",
+            "analyzer": "accent_analyzer",
+            "search_analyzer": "accent_analyzer",
             "fields": {
               "keyword": {
                 "type": "keyword",
                 "ignore_above": 256
+              },
+              "original": {
+                "type": "text"
               }
             }
           },
@@ -1125,7 +1138,14 @@
                 }
               },
               "title": {
-                "type": "text"
+                "type": "text",
+                "analyzer": "accent_analyzer",
+                "search_analyzer": "accent_analyzer",
+                "fields": {
+                  "original": {
+                    "type": "text"
+                  }
+                }
               },
               "type": {
                 "type": "object",
```